spec/requests/api/graphql_controller_spec.rb

```diff
@@ -10,7 +10,7 @@ def result_source_tenant(response_body)
   end
 
   context "querying source tenant" do
-    before { stub_const("ENV", "BYPASS_TENANCY" => nil) }
+    #before { stub_const("ENV", "BYPASS_TENANCY" => nil) }
```

Comment:
Not sure why this is here but it passes without it?

Reply:
I'll handle this in a separate PR

Reply:
I think old code tested presence of key in ENV

Reply:
Oh it looks like it is turned on/off as a `before {}` block but never restored so this might have been a sporadic test failure "fix"